README.md

```diff
@@ -11,17 +11,78 @@ Analyze, Annotate, Publish. Turn documents into data.
+5. You do not need to immediately start the docker-compose session for DocumentCloud until the Squarelet authentication service integration below is complete.
+6. Set `api.dev.documentcloud.org` and `minio.documentcloud.org` to point to localhost - `sudo echo "127.0.0.1 api.dev.documentcloud.org minio.documentcloud.org" >> /etc/hosts`
+7. Enter `api.dev.documentcloud.org/` into your browser - you should see the Django API root page. **Note that `api` is before `dev` in this service URL.**
+8. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their 2 repos as part of the full-stack application. 
+    - **Create Squarelet user:** 
+      - `inv sh` (not shell) then `./manage.py createsuperuser` in Squarelet (not DocumentCloud).
+      - Login using `inv shell` (not sh) in Squarelet
+      - Set your user as `is_staff`: (remove indentation after copying, if needed)
+      ```
+      tempUser = User.objects.all()[0]
+      tempUser.is_staff = True
+      tempUser.save()
+      ```
+   - **Setup frontend:** `make install` and `make dev` on the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) to start the frontend (used at end).
+9. Visit the Squarelet Django [admin page](http://dev.squarelet.local) with the `is_staff` user you created to configure the authentication integration. Follow the instructions as for the ["Squarelet Integration" on MuckRock](https://github.com/muckrock/muckrock/#squarelet-integration), except:
+   - When creating the OpenID as set out in those instructions, also add `Redirect URIs`: `http://api.dev.documentcloud.org/accounts/complete/squarelet` and `http://minio.documentcloud.org/accounts/complete/squarelet`. 
+   - Set Post-logout Redirect UI: `http://dev.documentcloud.org`
+   - Scopes (one per line): 
+      ```
+      read_user
+      read_organization
+      read_auth_token
+      ```
+   - Add a Client profile and set the `Webhook URL`: `http://api.dev.documentcloud.org/squarelet/webhook/`. The `Source` can remain "MuckRock" in the dropdown.
+   - You will have to look up **three** authentication-related configuration lines from your Squarelet instance to insert into DocumentCloud `.django` file (as if it was MuckRock). 
+   - Write their values into the `./.envs/.local/.django` file of the DocumentCloud repository, which should already be initialized from above.
+   - `Client ID` goes into `SQUARELET_KEY`
+   - `Client Secret` goes into `SQUARELET_SECRET`
+   - Additionally, get the value for `JWT_VERIFYING_KEY` by opening the Squarelet Django shell using `inv shell` and copying the `settings.SIMPLE_JWT['VERIFYING_KEY']` (remove the leading `b'` and the trailing `'`, leave the `\n` portions as-is)
```

Comment:
It would be nice to create an invoke task which automates this process

Reply:
From conversation with Mitch: Note that JWT piece is DocumentCloud specific and not in Muckrock codebase: it uses a different token type.